.github/workflows/pr_enclave_tls.yml

```diff
@@ -94,8 +93,13 @@ jobs:
           libsgx-dcap-default-qpl libsgx-dcap-ql libsgx-dcap-quote-verify \
           libsgx-enclave-common libsgx-launch libsgx-pce-logic libsgx-qe3-logic libsgx-quote-ex \
           libsgx-ra-network libsgx-ra-uefi libsgx-uae-service libsgx-urts sgx-ra-service \
-          sgx-aesm-service;
-        mv /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt'
+          sgx-aesm-service'
+
+    - name: install enclave-tls
+      run: docker exec $inclavare_test bash -c 'mv /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt;
+        mkdir -p /usr/share/enclave-tls;
+        mv /root/inclavare-containers/${{ matrix.tag }}/samples /usr/share/enclave-tls;
+        cp /usr/share/enclave-tls/samples/sgx_stub_enclave.signed.so ./'
```

Comment:
This pr mainly solves the path of enclave-tls changing problem, So It's better to change 
- `/usr/share/enclave-tls/samples`  to a ENV like `ENCLAVE_TLS_BINDIR`
       -  so `/usr/share/enclave-tls` can be written like `$(dirname %{ENCLAVE_TLS_BINDIR})`
- `/opt/enclave-tls` to a ENV `ENCLAVE_TLS_ROOTDIR`